Experiments/Experiments.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```

Comment:
These frameworks are in a limbo state. They'll become Swift packages shortly, but not until their dependencies are ready for it. When they'll be modules, their dependencies will be tracked in `Modules/Package.swift`. For the time being, they'll need to be defined individually in the projects.